### ISSUE TITLE
Clip-path - disable ability to remove or add points on responsive and hover state

### ISSUE THREAD

**Issue by Olekrut** [maxi-blocks/maxi-blocks#5230]
### Description

Clip-path - disable ability to remove or add points on responsive and hover state.

### Step-by-step reproduction instructions

1) add image or BG image 
2) enable clip-path ( polygon )
3) add few extra points
4) remove or add points on another BP or hover state

### Screenshots, screen recording, code snippet

https://jam.dev/c/49859d63-3db2-495b-8c34-c929286b5ecf

**Comment by Fybex** [comment]
What are the advantages of this choice? Simplification?

As I can see from the video, responsive and hover work correctly with different numbers of points, so I suggest leaving it up to the user whether he wants to use different numbers of points or not on the hover/responsive breakpoints. Maximum freedom is a good thing IMO.

**Comment by Olekrut** [comment]
I Agree that freedom in decisions is good, but Transition does not work with different amount of points, so we on porpose leave way for mistakes
